### PR TITLE
Fix ImageNet example in doc and add comparative data loading example

### PR DESCRIPTION
**Pull request by satyaog**
Add a comparative data loading example with PyTorch and ImageNet

### CODE REVIEW

**Comment by satyaog** [comment]
Hi @obilaniu, in the example I'm assuming that the validation set is included in the h.264 dataset. Let me know if that's right and what's the index where the data changes from originating of train to validation. I'll update the pr respectively.

**Comment by obilaniu** [comment]
@satyaog The last 100,000 examples are test. The previous 50,000 are validation. The first 1.2M are training.
